web/satellite/src/components/team/ProjectMembersArea.vue

```diff
@@ -104,9 +104,16 @@ export default class ProjectMembersArea extends Vue {
 
     /**
      * Returns team members of current page from store.
+     * With project owner pinned to top
      */
     public get projectMembers(): ProjectMember[] {
-        return this.$store.state.projectMembersModule.page.projectMembers;
+        const projectMembers = this.$store.state.projectMembersModule.page.projectMembers;
+        const projectOwner = projectMembers.find((member) => member.user.id === this.$store.getters.selectedProject.ownerId);
+        const nonOwners = projectMembers.filter((member) => member.user.id !== this.$store.getters.selectedProject.ownerId);
+
+        projectOwner && nonOwners.unshift(projectOwner);
```

Comment:
nit - a comment might help explain what this line is doing. Something like `// If the project owner exists, put it at the front of the members list`

another nit - Since you end up putting the projectOwner back into the "nonOwners" list, I don't think "nonOwners" is a good name for it (it ends up including the owner by the end of the function). I'd rather call it "projectMembersToReturn" or something. Maybe there's a better name though.

Reply:
I agree with the name change, couldn't think of anything better so went with your suggestion! Added the comment as well